### DISCUSSION TITLE
Bulk Charge

### DISCUSSION THREAD

**Discussion by mcconnon12** [swellstores/community#305] [Ideas]
Is bulk charging on the radar?

Currently if you set up your payments to charge later... You have to go into each order to charge it.

It would be nice if we could select all orders awaiting payment and charge them. Or at least 25 at a time. 

Is that something being considered?

Thanks,
Shawn

**Comment by ericingram** [comment]
It’s on the radar for sure. We’ve talked about it a number of times, but it would more likely be the domain of an App instead of a native feature. Should have more to announce on that topic in the next few months 😉
